docs/users/interact-with-gnokey.md

```diff
@@ -148,16 +149,16 @@ The `addpkg` subcommmand uses the following flags and arguments:
 - `-pkgdir` - local path where your is located
 - `-broadcast` - enables broadcasting the transaction to the chain
 - `-deposit` - a deposit amount of GNOT to send along with the transaction
-- `-gas-wanted` - the upper limit for units of gas for the execution of the
-  transaction
-- `-gas-fee` - amount of GNOTs to pay per gas unit
+- `-gas-wanted` - the upper limit for units of gas (optional, defaults to auto)
+- `-gas-fee` - amount of GNOTs to pay per gas unit (optional, defaults to auto)
```

Comment:
```suggestion
- `-gas-fee` - amount of GNOTs to pay (optional, defaults to auto)
```
It is not "per gas unit". The `-gas-fee` is the actual number of coins to be spent.